webapp/javascript/pages/ExploreView.tsx

```diff
@@ -0,0 +1,309 @@
+          activeTagProfile: undefined,
+        };
+
+      default:
+        return {
+          groupsData: [],
+          activeTagProfile: undefined,
+        };
+    }
+  };
+
+  const { groupsData, activeTagProfile } = getGroupsData();
+
+  const handleGroupByTagValueChange = (groupByTagValue: string) => {
+    dispatch(actions.setExploreViewGroupByTagValue(groupByTagValue));
+  };
+
+  const handleGroupedByTagChange = (value: string) => {
+    dispatch(actions.setExploreViewGroupByTag(value));
+  };
+
+  return (
+    <div className={styles.exploreView}>
+      <Toolbar hideTagsBar />
+      <Box>
+        <ExploreHeader
+          appName={appName}
+          tags={tags}
+          selectedTag={exploreView.groupByTag}
+          handleTagChange={handleGroupedByTagChange}
+        />
+        <TimelineChartWrapper
+          timezone={offset === 0 ? 'utc' : 'browser'}
+          data-testid="timeline-explore-page"
+          id="timeline-chart-explore-page"
+          timelineA={{ data: undefined }}
+          timelineB={{ data: undefined }}
+          timelineGroups={groupsData}
+          onSelect={(from, until) => dispatch(setDateRange({ from, until }))}
+          height="125px"
+          format="lines"
+        />
+      </Box>
+      <Box>
+        {appName.isJust && (
+          <Table
+            appName={appName.value}
+            groupsData={groupsData}
+            groupByTagValue={groupByTagValue}
+            handleGroupByTagValueChange={handleGroupByTagValueChange}
+          />
+        )}
+      </Box>
+      {groupByTagValue && (
+        <Box>
+          <FlamegraphRenderer
+            showCredit={false}
+            profile={activeTagProfile}
+            colorMode={colorMode}
+          />
+        </Box>
+      )}
+    </div>
+  );
+}
+
+export function calculateMean(arr: number[]) {
+  return arr.reduce((acc, b) => acc + b, 0) / arr.length;
+}
+
+export function calculateStdDeviation(array: number[], mean: number) {
+  const stdDeviation = Math.sqrt(
+    array.reduce((acc, b) => {
+      const dev = b - mean;
+
+      return acc + dev ** 2;
+    }, 0) / array.length
+  );
+
+  return stdDeviation;
+}
+
+function Table({
+  appName,
+  groupsData,
+  groupByTagValue,
+  handleGroupByTagValueChange,
+}: {
+  appName: string;
+  groupsData: TimelineGroupData[];
+  groupByTagValue: string | undefined;
+  handleGroupByTagValueChange: (groupedByTagValue: string) => void;
+}) {
+  return (
+    <>
+      <div className={styles.tableDescription}>
+        <span className={styles.title}>{appName} Descriptive Statistics</span>
+        <div>
+          <button>Export</button>
+          <button>Single</button>
+          <button>Comparison</button>
+          <button>Diff</button>
+        </div>
+      </div>
+      <table className={styles.exploreTable}>
+        <thead>
+          <tr>
+            <th>{groupsData[0]?.tagName === '*' ? 'App' : 'Tag'} name</th>
```

Comment:
What is this doing?

Reply:
added few comments, and refactored `*` comparison (not only in this component)